src/pages/DetailMemoryPage.js

```diff
@@ -0,0 +1,68 @@
+import styled from 'styled-components/macro'
+import Button from '../components/Button'
+import PropTypes from 'prop-types'
+import MemoriesIcon from '../assets/images/icons/galery.png'
+
+DetailMemoryPage.propTypes = {
+  image: PropTypes.node,
```

Comment:
This should be a string (as it is a url).

Reply:
It`s a local image-file.